run/grpc-ping/README.md

```diff
@@ -0,0 +1,119 @@
+# Cloud Run gRPC Ping Sample Application
```

Comment:
We generally try to avoid having in-depth readmes for samples, instead linking to the tutorial they are for. If that tutorial doesn't exist yet, we can keep this. But, eventually, this may cause a maintenance issue if we forget to update it.

Reply:
In this case the sample will be used in a how-to doc, I'm including the README because the instructions for build, deploy, and usage are fairly complicated. I'm using services because end-to-end tests are necessary to prove auth and gRPC serving.